src/components/Results.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from "react";
+import TableHead from './TableHead';
+import TableBody from './TableBody';
+import * as FakeBookings from "../data/fakeBookings.json";
+
+
+class Results extends React.Component {
+	constructor(props) {
+		super(props)
+
+		this.state = {
+			data: [],
+			isFetched: false
+		}
+
+		this.checkIfFetched = this.checkIfFetched.bind(this);
```

Comment:
Because we are using create-react-app, we can use the class properties syntax and avoid the `.bind` call here (it's a bit ugly 🙂). So it would look like:

```js
checkIfFetched = (event) => {
  ...
}
```

Reply:
not sure what you mean, sorry

Reply:
So your code here is valid. But having to add `this.whatever = this.whatever.bind(this)` for every method in your code is a bit of a pain.

So there is a new way to do the same thing in JavaScript. This new syntax is enabled because the homework was created with create-react-app.

If you remove the `.bind` line above, then change the `checkIfFetched` method to use the (slightly different) syntax that I pasted above, it's a bit nicer to read and you don't have extra lines. It's minor but if you have lots of methods in a component it can make it easier to read.